src/article.py

```diff
@@ -0,0 +1,22 @@
+from typing import Optional 
+from dataclasses import dataclass, field
+from dataclasses_json import dataclass_json
+
+import article_parser
+
+
+@dataclass_json
+@dataclass
+class Article:
+    url: str # only this field is mandatory
+    # title and content are instanciated by the url parser
+    title: Optional[str] = None 
+    content: Optional[str] = None
+    # suggested_title is instanciated by the LLM
+    suggested_title: Optional[str] = None 
```

Comment:
i would remove code unused (suggested, meta_data) for now. When we do need it we might need something slightly different anyway

Reply:
Done !